ubuntudesign/www.ubuntu.com
# lint and tidy of desktop scss

## Done

* linted and tidied up the _desktop.scss

## QA

1. go to every page in the /desktop section and see that it looks the same as live
